### ISSUE TITLE
Download onto non default directory broken

### ISSUE THREAD

**Issue by Qik000** [ValveSoftware/steam-for-linux#5542]
#### Your system information

* Steam client version (build number or date):  Ubuntu 18.04 LTS
* Distribution (e.g. Ubuntu): Ubuntu 18
* Opted into Steam client beta?: [Yes/No] NO
* Have you checked for system updates?: [Yes/No] YES

#### Please describe your issue in as much detail as possible:
Steam for some reason downloads over and over CSGO and other games instead of launching them when trying to install in non-default directory. Sometimes it recognizes it just right, but sometimes i just cant get it to admit csgo`s existance on my hdd

#### Steps for reproducing this issue:

1. Add a directory on a diffrent drive to steam
2. Try to download and install a game in that directory

**Comment by kisak-valve** [comment]
Hello @Qik000, what filesystems and paths are you using with your Steam libraries?

**Comment by Qik000** [comment]
@kisak-valve the non-default one is on NTFS, and as i said, it sometimes works, sometimes not, depending on how it feels

**Comment by kisak-valve** [comment]
Possibly related to #4800.

**Comment by sizeofbool** [comment]
@Qik000 @verdegr check if you mounted a partition used for external path with proper permissions.
Valve can't get into your OSs and change internal system settings without user's consent and authorization.
Default settings for "ntfs-3g" driver are restrictive, so this "bug" won't be fixed, because you have to mount NTFS filesystem properly by yourself.

Example line for mounts file "/etc/fstab":
```
UUID=1BF5E1890BC91113 /media/WindowsPartition ntfs-3g auto,permissions,uid=1000,gid=1000 0 0
```
or
```
/dev/sda1 /media/WindowsPartition ntfs-3g auto,permissions,uid=1000,gid=1000 0 0
```
Where "uid=1000,gid=1000" are effective IDs of your user.

**Comment by Qik000** [comment]
@sizeofbool everything is all right, since it can pickup csgo from hdd sometimes, its just not consistant

**Comment by kisak-valve** [comment]
Hello @Qik000, @verdegr, are either of you still experiencing this issue on an up to date system?

**Comment by kisak-valve** [comment]
Closing pending feedback.

**Comment by Qik000** [comment]
> Hello @Qik000, @verdegr, are either of you still experiencing this issue on an up to date system?

Yes, but only with some games. Seems to choose at random with which games to have a problem with
